.github/workflows/codeql_analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+ # to commit it to your repository.
+ #
+ # You may wish to alter this file to override the set of languages analyzed,
+ # or to provide custom queries or build logic.
+ #
+ # ******** NOTE ********
+ # We have attempted to detect the languages in your repository. Please check
+ # the `language` matrix defined below to confirm you have the correct set of
+ # supported CodeQL languages.
+ #
+ name: "CodeQL"
+
+ on:
+   push:
+     branches: [ main, development ]
```

Comment:
I don't know if we really wnat CodeQL running on PRs to dev - I think main is sufficient? It takes for ever and as not brought up anyting for us yet.